### DISCUSSION TITLE
Welcome to Solodit feature discussions!

### DISCUSSION THREAD

**Discussion by hans-cyfrin** [solodit/feedback#1] [New Feature Requests]
## 👋 Welcome!
  We’re using Discussions as a place to gather requests for new features on [Solodit](https://solodit.xyz/). We encourage you to:
  * Request features that you would like to see added.
  * Engage in discussions to prioritize and determine the most essential features to be implemented first.

  To get started, create a new discussion under the category of "New Feature Requests".

**Comment by IAm0xAxe** [comment]
I've just submitted a discussion about "SLOCs". ^_^

**Reply by hans-cyfrin** [reply]
Thank you for your suggestion. We will add this feature to our backlog.

**Comment by V1C70RYG0D** [comment]
add a feature to track my checked finding in a separate tab , where i can track my progress , also category wise

**Comment by hans-cyfrin** [comment]
For future users, please create a new discussion at [New Feature Requests](https://github.com/orgs/solodit/discussions/categories/new-feature-requests).
